Doc/library/importlib.rst

```diff
@@ -1719,6 +1719,27 @@ To import a Python source file directly, use the following recipe
   spec.loader.exec_module(module)
 
 
+Implementing lazy imports
+'''''''''''''''''''''''''
+
+The example below shows how to implement lazy imports::
+
+  import sys
+  import importlib.util
+  def lazy_import(name):
+    spec = importlib.util.find_spec(name)
```

Comment:
Hi @nanjekyejoannah 

I like the example but could you use an indent of 4 spaces to be conform with the other examples?

Thank you,